apps/docs/pages/guides/database/large-datasets.mdx

```diff
@@ -0,0 +1,121 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'database',
+  title: 'Importing into Supabase',
+  description: 'Importing Large Datasets into Supabase.',
+  subtitle: 'Importing Large Datasets into Supabase.',
+  sidebar_label: 'FUNDAMENTALS',
+}
+
+Importing large datasets requires careful planning and execution to ensure the process is efficient, reliable, and doesn't disrupt your application. In this guide, we cover various best practices and techniques for importing datasets, while also highlighting common pitfalls to avoid.
+
+## **You have multiple options for importing your data into Supabase**
```

Comment:
For headings, I don't think you'll need `**` - they will be formatted accordingly by default